.circleci/config.yml

```diff
@@ -91,6 +91,17 @@ jobs:
       - run: stack exec regression
       - run: stack exec gaussian_process
       - run: stack exec vae
+  nix-build:
+    docker:
+      - image: "nixos/nix:latest"
+    steps:
+      - run: nix-env -i git
+      - checkout
+      - run: git submodule init && git submodule update
+      - run: nix-env -iA cachix -f https://cachix.org/api/v1/install
+      - run: cachix use hasktorch
+      - run: cd deps;nix-shell --pure -p python37.pkgs.pyyaml --run "./get-deps.sh -s"
```

Comment:
I believe you don't need this line. the dependencies come all through nix

Reply:
The test of codegen stops not to find yaml-files.
So I run the python-command, but the error is remained.(I do not know the reason.)

Reply:
I'm not sure how line 103 and this problem can be related :/
I was just remarking that it should not be necessary to install any dependencies from the `./deps` folder because `libtorch`, `mkl`, and so on are pulled in through the nix derivations I made without further ado.

Reply:
never mind, I could reproduce the issue:

Reply:
I think I skipped the tests for `codegen` before, and that's why I never ran into this.

It's a bit wasteful to download all of pytorch just for a `yaml` file, isn't it?

Reply:
Yes, it is. But we need some python codes and header-files to generate yaml files.